.npmignore

```diff
@@ -0,0 +1,15 @@
+.github/
+build-demo/
+.DS_Store
+cypress/
+demo/
+node_modules/
+scripts/
+src/
+.eslintrc.yml
+.gitignore
+.travis.yml
+*.sh
+cypress.json
+demos.md
+error.html
```

Comment:
Can we just use the `files` part of the `package.json`?  Maybe I misunderstand it, but I think it's a little cleaner.

Reply:
From the travis CI docs it is unclear to me whether that will work, but I agree that it is much cleaner, so I will remove and if it doesn't work we can always add the `npmignore` back.

https://docs.travis-ci.com/user/deployment/npm/#note-on-gitignore

The reason I am unsure is that it says `Notice that npm deployment honors .gitignore if .npmignore does not exist` but in `.gitignore` we ignore the `build-lib/` directory which is what we want to publish. But the local `npm publish` command works fine without `.npmignore`

Reply:
Yikes, that kind of scares me....ok, do whatever you feel is best then.